driver_go18_test.go

```diff
@@ -468,3 +469,100 @@ func TestContextCancelBegin(t *testing.T) {
 		}
 	})
 }
+
+func TestContextBeginIsolationLevel(t *testing.T) {
```

Comment:
nice test👍 

but, this test can be written more concisely, can't it?
for example

``` go
tx1, err := dbt.db.BeginTx(ctx, &sql.TxOptions{
    Isolation: sql.LevelRepeatableRead,
})
tx2, err := dbt.db.BeginTx(ctx, &sql.TxOptions{
    Isolation: sql.LevelReadCommitted,
})

row := tx2.QueryRowContext(ctx, "SELECT COUNT(*) FROM test")
// check row.

_, err = tx1.ExecContext(ctx, "INSERT INTO test VALUES (1)")

row = tx2.QueryRowContext(ctx, "SELECT COUNT(*) FROM test")
// check row.
```